string/readme.md

```diff
@@ -15,4 +15,12 @@ A string is a class that contains a char array, but automatically manages it for
 
 * Convert string to lowercase and uppercase ---->[C++](/Code/C++/string_to_lowercase_and_uppercase.cpp)
 * Revere a string using Stack ---->[C++](/Code/C++/reverse_a_string_using_stack.cpp)
+
+
+* Checking for Anagram ---->[Java](/Code/Java/Anagram.java)
+
+* Reverse a string ---->[C++](/Code/C++/reverse_string.cpp)
```

Comment:
Plz have a look of what changes have you done via Files changed option and make changes appropriately.